obspy/clients/nrl/client.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Client for accessing the Nominal Response Library (http://ds.iris.edu/NRL/).
+
+:copyright:
+    Lloyd Carothers IRIS/PASSCAL, 2016
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import codecs
+import io
+import os
+import sys
+import warnings
+
+import requests
+
+import obspy
+from obspy.core.compatibility import configparser
+from obspy.core.inventory.response import ResponseStage
+from obspy.core.inventory.util import _textwrap
+
+
+# Simple cache for remote NRL access. The total data amount will always be
+# fairly small so I don't think it needs any cache eviction for now.
+_remote_nrl_cache = {}
+
+def _cleanup_response(response):
```

Comment:
I suppose I'm looking at this from the other end:
Why add clutter to the code?
As not the author of the response, we don't own that clutter.
They act as a placeholder.
Likely put there for a specific reason by the author e.g. DL gain is set to unity/1.

 While this function looks solid and comprehensive, the improvement is negligible, and complexity is added. ( Unthought of corner cases, strange responses becoming stranger, debugging: another section of 'does the problem lie here'... the same of any amount of added code.)

 Perhaps a way of respecting the response's author; Keeping the response as intact as possible. If I think there is an issue with a response and I take it to the author, seeing a different number of stages, thier reaction will probably include 'what else has changed?' and troubleshoot will then require an added step of comparison of the decluttered to the original, and confirming the dropped stages, and comparing the same stages with different indexes of the stages following the dropped ones. 

 There can be some information in those stages, e.g. the rt-130 mentioned on the PR.  This drops stage 2 which is the settable gain that can be 1 or 32 for this model. This placeholder is nice to see that the DL was set with a gain of 1. Also easier to compare 2 responses from this DL with different gains, and all the coressponding stages match by stage #.